girder/utility/assetstore_utilities.py

```diff
@@ -60,6 +59,15 @@ def getAssetstoreAdapter(assetstore, instance=True):
         return cls
 
 
+def setAssetstoreAdapter(storeType, cls):
+    global assetstoreTable
+
+    if storeType not in assetstoreTable:
+        raise Exception('Illegal assetstore type code: "%s"' % (storeType))
```

Comment:
Maybe in 2.0, we'd leave it around for now for backward compatibility, but this would be the "recommended" way moving forward.